packages/react-components/src/organisms/ResearchOutputExtraInformationCard.tsx

```diff
@@ -91,9 +93,11 @@ const TeamCreateOutputExtraInformationCard: React.FC<TeamCreateOutputExtraInform
     }, [type, fetchResearchTags]);
 
     useEffect(() => {
-      onChangeMethods([]);
-      onChangeOrganisms([]);
-    }, [type, onChangeMethods, onChangeOrganisms]);
+      if (!isEditMode) {
+        onChangeMethods([]);
+        onChangeOrganisms([]);
+      }
```

Comment:
This resets the methods and organisms when the type changes. We probably want this in edit mode.